pyhf/pdf.py

```diff
@@ -450,3 +451,130 @@ def logpdf(self, pars, data):
     def pdf(self, pars, data):
         tensorlib, _ = get_backend()
         return tensorlib.exp(self.logpdf(pars, data))
+
+
+class Workspace(object):
+    def __init__(self, spec, **config_kwargs):
+        self.spec = spec
+
+        self.schema = config_kwargs.pop('schema', 'workspace.json')
+        # run jsonschema validation of input specification against the (provided) schema
+        log.info("Validating spec against schema: {0:s}".format(self.schema))
+        utils.validate(self.spec, self.schema)
+
+        self.measurement_names = []
+        for measurement in self.spec.get('measurements', []):
+            self.measurement_names.append(measurement['name'])
+
+    # NB: this is a wrapper function to validate the returned measurement object against the spec
+    def get_measurement(self, **config_kwargs):
+        """
+        Get (or create) a measurement object using the following logic:
+
+        1. if the poi name is given, create a measurement object for that poi
```

Comment:
All these items in the `get_measurement` docstring are prefixed by `1.` Is this intentional, or were these supposed to be `1.`, `2.`, ...?

Reply:
I didn't check how the documentation rendered these. I generally assumed markdown rendering.